src/m365/entra/commands/group/group-add.ts

```diff
@@ -101,32 +101,30 @@ class EntraGroupAddCommand extends GraphCommand {
         }
 
         if (args.options.ownerIds) {
-          const ids = args.options.ownerIds.split(',').map(i => i.trim());
-          if (!validation.isValidGuidArray(ids)) {
-            const invalidGuid = ids.find(id => !validation.isValidGuid(id));
-            return `'${invalidGuid}' is not a valid GUID for option 'ownerIds'.`;
+          const isValidGUIDArrayResult = validation.isValidGuidArray(args.options.ownerIds);
```

Comment:
I think I disagree on the implementation here @Jwaegebaert. While moving the action of splitting the string to the util function seems like a good way to cut down on lines of code, it also makes the function less flexible. If we would want to validate a true array which is not coming from an options string, or a string that is not separated by comma's, we would now not be able to do that. And also, the naming is vague. We're talking about validating an arra (`isValidGuidArray`), while we are validating a string value.

Reply:
It makes the function less flexible but is it a common scenario that we'll validate GUIDs further down the line? If that is the case, we can make the function more versatile by allowing a string or a GUID array. I don't think this is a use case we'll see often so it could be something to keep in mind for the future. Certainly when we're looking more into `zod`. 

Regarding naming, fully agreed. Maybe something like `areValidGuids` or `validateGuids`

Reply:
You're right that we're moving to zod in some time. I'm still not a fan of this implementation, but as it's probably swapped for zod anyway, and the upn function is implemented in the same way, I'll overlook it for now. 👍